src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -327,9 +327,10 @@ int MPIDI_OFI_control_handler(int handler_id, void *am_hdr,
                               MPIDIG_am_target_cmpl_cb * target_cmpl_cb, MPIR_Request ** req);
 int MPIDI_OFI_control_dispatch(void *buf);
 void MPIDI_OFI_index_datatypes(void);
+
 void MPIDI_OFI_index_allocator_create(void **_indexmap, MPL_memory_class class);
```

Comment:
Here is an empty line change. Can you please remove it ?

Reply:
Will do.